indra/tests/test_mesh.py

```diff
@@ -115,3 +115,17 @@ def test_mesh_mapping():
         'D000077143'
     assert mesh_client.get_db_mapping('D000077143') == \
         ('CHEBI', 'CHEBI:4672')
+
+
+def test_supplementary_to_primary():
+    from indra.statements.validate import assert_valid_id
+    for supp, primaries in mesh_client.mesh_supp_to_primary.items():
+        for primary in primaries:
+            assert_valid_id('MESH', primary)
+    mesh_client.get_primary_mappings('C009879') == ['D011140']
+    mesh_client.get_primary_mappings('C114158') == ['D005063', 'D009942']
```

Comment:
I was under the impression that there were things in the supplement that didn't have a corresponding primary record. Is this not true?

Reply:
I actually didn't find any examples of that, all the supplementary concepts are mapped to one or more primary term.

Reply:
Fascinating! I will have to look into this and definitely improve the pyobo mesh importer

Reply:
Yeah, that's actually something they enforce when they make supplementary concepts. Sometimes the mappings are to something much higher up in the ontology, but there's always at least one mapping.

```
Supplementary Records
Supplementary Records, also called Supplementary Chemical Records(SCRs), are used to index chemicals, drugs, and other concepts such as rare diseases for MEDLINE and are searchable by Substance Name [NM] in PubMed. Unlike Descriptors, SCRs are not organised in a tree hierarchy. Instead each SCR is linked to one or more Descriptors by the Heading Mapped To (HM) field in the SCR. They also include a Indexing Information (II) field field that is used to refer to other descriptors that are from related topics. SCRs are created daily and distributed nightly Monday-Thursday. There are currently over 230,000 SCR records with over 505,000 SCR terms. Like all MeSH records, SCRs are searchable in the MeSH Browser. Four classes of SCRs exist. 
```
https://www.nlm.nih.gov/mesh/intro_record_types.html

Reply:
Oh that makes more sense